lib/src/main/resources/db/changelog/social-rdbms.db.changelog-1.0.0.xml

```diff
@@ -205,7 +205,7 @@
         <column name="REGISTRATION" type="TINYINT">
           <constraints nullable="false"/>
         </column>
-        <column name="DESCRIPTION" type="LONGTEXT"/>
+        <column name="DESCRIPTION" type="NVARCHAR(1000)"/>
```

Comment:
Did you make some tests with the others databases to be sure there is no regression ?